src/main/java/net/dv8tion/jda/api/entities/ThreadMember.java

```diff
@@ -22,32 +22,62 @@
 import javax.annotation.Nonnull;
 import java.time.OffsetDateTime;
 
-//TODO-v5: docs
+/**
+ * A {@link ThreadMember} represents a {@link Member Member's} membership in a Thread.
+ * <br>ThreadMembers are subscribed to {@link ThreadChannel Threads} and receive updates for them, like new message and thread changes.
+ * Only subscribed members are shown in a thread's sidebar.
+ */
 public interface ThreadMember extends IMentionable
 {
+    /**
+     * Returns the {@link net.dv8tion.jda.api.JDA JDA} instance of this threadmember.
+     *
+     * @return the corresponding JDA instance
+     */
     @Nonnull
     JDA getJDA();
 
+    /**
+     * The {@link Guild} containing this {@link ThreadMember ThreadMembers} and it's {@link ThreadChannel}.
+     *
+     * @return The {@link Guild} containing this {@link ThreadMember ThreadMembers} and it's {@link ThreadChannel}.
+     */
     @Nonnull
     Guild getGuild();
 
+    /**
+     * The {@link ThreadChannel} this threadmember is subscribed to.
```

Comment:
```suggestion
     * The {@link ThreadChannel} this thread member is subscribed to.
```

Reply:
Applies to all `threadmember` instances. When lowercased they should be separated.
Or PascalCase them. Either way.